web-api/src/storage/elastic.rs

```diff
@@ -314,11 +320,16 @@ impl storage::Document for Storage {
                 "num_candidates": params.num_candidates,
                 "filter": {
                     "bool": {
+                        "must": {
+                            "range": {
+                                "publication_date": range,
```

Comment:
:art: the macro allows for trailing commas, but it is invalid json syntax and we should avoid it
```suggestion
                                "publication_date": range
```